### DISCUSSION TITLE
2.9.0-preview2

### DISCUSSION THREAD

**Discussion by arianneorpilla** [arianneorpilla/jidoujisho#340] [Announcements]
This is a **preview release** for  **2.9**. Please refer to the [**releases page**](https://github.com/lrorpilla/jidoujisho/releases) for a full timeline of changes. 

# 2.9.0 Preview 2
- @Aegyo: Fixes an issue where using the app with newer versions of AnkiDroid (2.17) would have issues with interacting with the new backend (#339).
- @Aegyo: Migaku Dictionary imports now include the pronunciation field as the reading (#338).

### Known Issues and Notes
- Jellyfin has been reported to have changed the parameters they pass to external players for subtitles. Please use an older version of Jellyfin Android (2.6.0-beta1) before this change for the time being.
- I'm reading everyone's feedback and am taking an extended hiatus at the moment. The next stable release is not yet ready. Please be patient.

# 📲 Which file should I download?
As a rule of thumb, `arm64` should work for most modern mobile devices. `armeabi` should work on older devices. Generally, desktops and emulators should use `x86_64`.

Depending on which device you are using, download the appropriate `APK` by expanding the `Assets` drop-down below and install on your mobile phone by launching the `APK` with your file explorer.

<hr /><em>This discussion was created from the release <a href='https://github.com/lrorpilla/jidoujisho/releases/tag/2.9.0-preview2'>2.9.0-preview2</a>.</em>

**Comment by Aegyo** [comment]
Is it possible these build files don't have the migaku dictionary change?

I installed the `jidoujisho_yuuna_2.9.0_preview2-arm64.apk`, cleared dictionaries and re-imported, but I still don't see pronunciations. Building the app myself from the latest main branch does show the pronunciations however.

**Reply by arianneorpilla** [reply]
My bad. I'll build and reupload files from latest `main` now.

**Reply by Aegyo** [reply]
Working for me now, thank you!
